src/components/Math/Math.jsx

```diff
@@ -0,0 +1,93 @@
+import React, { Component } from 'react'
+import { TextField, Button , Select } from '@material-ui/core';
+import { pStyle, divStyle } from '../TextField/style';
+
+
+export default class Math extends Component {
+    constructor(props){
+        super(props);
+        this.state={
+            first: 0,
+            second: 0,
+            operator: null,
+            result:null
+        }
+    }
+    // const { first, second, operator } = this.state;
+    handleResultButton = (event) => {
+        event.preventDefault();
+    // eslint-disable-next-line no-unused-vars
+        let result = this.state.result;
```

Comment:
Use Destructuring concept and remove all console.log

Reply:
okay Sir